cache_test.go

```diff
@@ -0,0 +1,74 @@
+package cache
+
+import (
+	"fmt"
+	"net/http/httptest"
```

Comment:
@easonlin404 `net/http/httptest` not available @ go1.6, need to edit `.travis.yml`